2022/project_idea.md

```diff
@@ -1,106 +1,116 @@
+But at the same time, explaining how to use a PPL to someone who doesn't know 
+anything about Bayesian modeling is also a fool's errand. 
+Any modeling without visualization, diagnostics and model checking is 
 doomed from the start.
-**We need to teach all these skills at the same time in an organized and coherent manner**
+**We need to teach all these skills at the same time in an organized and coherent manner**.
 
-In fact, this is already what is happening. For several years already,
-all books in Bayesian modeling use and explain computational tools,
-they very often have a "paired" GitHub repo with the code used in the book,
-_and the most popular books are "translated" to other PPLs and programming languages_.
+In fact, this is already what is happening. For several years now, 
+all books in Bayesian modeling uses and explain computational tools, 
+they very often have a "paired" GitHub repo with the code used in the book, 
+_and the most popular books are "translated" to other PPLs and programming languages._ 
 To give one example, [Statistical Rethinking](https://xcelab.net/rm/statistical-rethinking/)
-has had their code examples translated into at least 3 programming languages
-(R, Python and Julia) and multiple PPLs. The book homepage lists 10 such translations.
-A 2 minute search to see if the book had been translated into Spanish only
+has had their code examples translated into at least 3 programming languages 
+(R, Python and Julia) and multiple PPLs. The book homepage lists 10 such translations. 
+A 2 minute search to see if the book had been translated into Spanish only 
 brought up Spanish bookstores selling the original English version of the book.
 
 ## About the project
-Given's ArviZ position within the PPL ecosystem; some of the members of our team
-are also members of PPL development teams. We would like to create an open, collaborative, online
-and multi-language (in all its meanings!) educational resource about the Bayesian workflow.
+Given ArviZ's position within the PPL ecosystem, some of the members of our team 
+double as members of PPL development teams. We would like to create an open, collaborative, online 
+and multi-language (in all its meanings!) educational resource about the Bayesian workflow. This resource will cover the various compatible libraries that can be used to create Bayesian workflow with sample visualizations and codes.
 
-The goal of the project is to set up all the infrastructure for such a resource to
-avoid duplication, and once this is achieved guide the ArviZ educational lead
+The goal of the project is to set up all the infrastructure for such a resource to 
+avoid duplications like the one encountered above, and once this is achieved, guide the ArviZ educational lead 
 so they can start writing content.
 
-## Project’s scope
-The Arviz documentation project will
-* Evaluate multiple alternatives to generate this Bayesian workflow resource.
+### Project’s scope
```

Comment:
```suggestion
## Project’s scope
```

To match the original hierarchy which came from the gsod template. "about the project" and "project scope" should be at the same level, not one inside the other.